_pytest/main.py

```diff
@@ -29,7 +29,7 @@
 
 def pytest_addoption(parser):
     parser.addini("norecursedirs", "directory patterns to avoid for recursion",
-        type="args", default=['.*', 'CVS', '_darcs', '{arch}', '*.egg'])
+        type="args", default=['.*', 'build', 'dist', 'CVS', '_darcs', '{arch}', '*.egg'])
```

Comment:
That's covered by `.*` (which is a fnmatch glob, not a regex)

Otherwise you'd find pytest finding your code in `.tox` all the time too!